packages/core/core/src/public/Bundle.js

```diff
@@ -168,20 +184,38 @@ export class NamedBundle extends Bundle implements INamedBundle {
   #bundleGraph; // BundleGraph
 
   constructor(
+    sentinel: mixed,
     bundle: InternalBundle,
     bundleGraph: BundleGraph,
     options: ParcelOptions,
   ) {
+    super(sentinel, bundle, bundleGraph, options);
+    this.#bundle = bundle; // Repeating for flow
+    this.#bundleGraph = bundleGraph; // Repeating for flow
+  }
+
+  static get(
```

Comment:
This resolves a bug that caused `Bundle`s to be returned instead of `NamedBundle`s as the `super()` call found the memoized `Bundle` and made that `this`.